wandb/verify/util.py

```diff
@@ -0,0 +1,403 @@
+import getpass
```

Comment:
I think you'll need to wire this up to the codemod that strips type annotations for py27.

Reply:
Hmm, if I am seeing correctly, it might be wired up. `apikey.py` and `wandb_settings.py` both use it and are codemodded.